Client/sdk/game/CPed.h

```diff
@@ -277,4 +286,6 @@ class CPed : public virtual CPhysical
     virtual CPedIKSAInterface*      GetPedIKInterface() = 0;
     virtual void*                   GetPedNodeInterface(std::int32_t nodeId) = 0;
     virtual std::unique_ptr<CPedIK> GetPedIK() = 0;
+
+    virtual void GetAttachedSatchels(std::vector<SSatchelsData> &satchelsList) = 0;
```

Comment:
noexcept?

Reply:
I'm not sure about this. Theoretically, there is a chance that an exception will occur here, for example, during vector allocation, and the program might terminate abruptly without providing any specific information about what went wrong. Adding noexcept to this function seems impractical to me because it implies that the function guarantees not to throw any exceptions. However, operations on std::vector, such as allocation and reallocation, can potentially throw exceptions if memory allocation fails. Therefore, I prefer not to mark this function as noexcept